app/controllers/v0/sessions_controller.rb

```diff
@@ -98,6 +98,12 @@ def user_login(saml_response)
       if user_session_form.valid?
         @current_user, @session_object = user_session_form.persist
         set_cookies
+        # track users who need to re-login on MHV
+        unless @current_user.mhv_correlation_id
+          StatsD.increment(
+            V1::SessionsController::STATSD_MHV_COOKIE_NO_ACCOUNT_KEY
```

Comment:
Doesn't seem right to create dependencies between `V0` & `V1`. Forgive me if we've already crossed this bridge and I'm uninformed.

Reply:
No worries; I'll duplicate this in V0—it's just a constant for the stats tracking